mapcomposer/app/static/externals/gxp/src/script/plugins/WMSSource.js

```diff
@@ -510,21 +512,55 @@ gxp.plugins.WMSSource = Ext.extend(gxp.plugins.LayerSource, {
         return compatibleProjection;
     },
     
+    /** private: method[switchWmsVersion]
+     *  returns the WMS version alternative to the one passed as parameter
+     */
+    switchWmsVersion: function(wmsVersion) {
+    	var WMS_VERSIONS = {
+    		"1.1.1": "1.3.0",
+    		"1.3.0": "1.1.1"
+    	};
+    	
+    	return WMS_VERSIONS[wmsVersion];
+    },
+    
     /** private: method[initDescribeLayerStore]
      *  creates a WMSDescribeLayer store for layer descriptions of all layers
      *  created from this source.
      */
     initDescribeLayerStore: function() {
-        var req = this.store.reader.raw.capability.request.describelayer;
-        if (req) {
-            this.describeLayerStore = new GeoExt.data.WMSDescribeLayerStore({
-                url: req.href,
-                baseParams: {
-                    VERSION: this.store.reader.raw.version,
-                    REQUEST: "DescribeLayer"
-                }
-            });
-        }
+    	var DEFAULT_WMS_VERSION = "1.1.1";
+    	var url;
+    	var version;
+    	var baseParams = {
+    			SERVICE: "WMS",
+    			REQUEST: "DescribeLayer"
+    		};
+    	
+    	if (this.useCapabilities === true) {
+    		var describeLayerRequest = this.store.reader.raw.capability.request.describelayer;
+    		var wmsVersion = this.store.reader.raw.version;
+    		
+    		if (describeLayerRequest) {
+    			url = describeLayerRequest.href;
+    			version = wmsVersion; 
+    		} else {
+    			url = this.url;
+    			version = this.switchWmsVersion(wmsVersion);
```

Comment:
I would make more explicit that in this case we are choosing the DEFAULT_WMS_VERSION and not switching to the other version.